src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -931,55 +965,12 @@ protected void UpdateState(Touch touchInfo = null)
         /// <since_tizen> 6 </since_tizen>
         private void Initialize()
         {
-            var style = (ButtonStyle)Style;
-
-            Extension = style.CreateExtension();
-
-            CreateComponents();
-
             EnableControlStatePropagation = true;
-
-            if (ButtonOverlayImage != null)
-            {
-                Add(ButtonOverlayImage);
-                ButtonOverlayImage.ApplyStyle(style.Overlay);
-            }
-
-            if (ButtonIcon != null)
-            {
-                Add(ButtonIcon);
-                ButtonIcon.ApplyStyle(style.Icon);
-                ButtonIcon.Relayout += OnIconRelayout;
-            }
-
-            if (null != ButtonText)
-            {
-                Add(ButtonText);
-                ButtonText.ApplyStyle(style.Text);
-            }
-
+            iconRelativeOrientation = IconOrientation.Left; 
```

Comment:
Button.ApplyStyle() is called before calling Button.Initialize().
So this over-write ButonStyle's IconOrientation value always.
Can you move this line to line297 or somewhere valid?

Reply:
I found this issue by running this [sample](https://github.com/dalihub/nui-demo/tree/linux/wearable-samples/Button/ButtonWithOverlayAnimation).

You can check valid screen-shot in README.md.

Reply:
Fixed